tests/devapps/Net70TestApp/Net70TestApp.csproj

```diff
@@ -0,0 +1,60 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworkNet70>net7.0</TargetFrameworkNet70>
```

Comment:
I think it's ok to just add net7.0-windows target to NetCoreTestApp. The core net7 code is the same, -windows suffix just allows to use Windows APIs.

Reply:
I agree, let's reuse an existing console app. You can even merge the .NET classic and the .NET core console apps if you wish.

Reply:
You might need to reference net7-windows10.0.17763.0 and not just net7-windows, but do give it a try.